src/msgpack/Convertes/ArrayConverter.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace TarantoolDnx.MsgPack
+namespace TarantoolDnx.MsgPack.Convertes
```

Comment:
It does not work for links and for folder on level above than project file. And we can't have them in one folder, because they have drastically different project.json files.

If you care, please show how to do it.